### ISSUE TITLE
Improvement: SAVE TO WAV

### ISSUE THREAD

**Issue by iz4tow** [lwvmobile/dsd-fme#163]
Good evening,
could be possible to write a wav file also for BP and/or Encrypted tracks in a future release?
Thanks

**Comment by lwvmobile** [comment]
I'll mull it over, see if there is a way to tie it into the source audio monitor code without causing any issues or making me re-write a chunk of that code to accommodate it. You can also save symbol capture bins that can be played back by using the `-c capture.bin` option, and play them back with `-i capture.bin`

**Comment by iz4tow** [comment]
Good afternoon,
I tried `dsd-fme -i test.wav -c test.bin ` and `dsd-fme  -i test.wav -b 1 -c test.bin`. But in both case the bin file is empty.
The difference, of course, is that with `-b 1` options dsd-fme will reproduce the audio but data in bin file

**Comment by lwvmobile** [comment]
I just tried the same thing you did, and it worked for me. I decoded a wav file input and output it to a bin file, and was then in turn able to use the bin as input and get the same decode. Not sure what you did differently, but also note that a 'small' bin file is not an empty bin file.

**Comment by iz4tow** [comment]
I'm not looking at the file size, I tried `cat` to look into bin file and it's empty.
I can provide the wav track if u want to try
Thanks

**Comment by lwvmobile** [comment]
Did you consider just trying to playback the bin file with 
`dsd-fme -i test.bin`

I ran a cat on my own bin files, and it doesn't show anything, its not in a format that cat will reveal.

**Comment by iz4tow** [comment]
wow! What a moron I was, using hexdump the bin file is not empty!
Thanks

**Comment by lwvmobile** [comment]
Alrighty, I added raw signal wav file saving (SB16LE 48k/1) as an option to v2.1b and audio_work branches. Its not available in the precompiled binary windows version yet, but next time I update it, it'll be in that as well. 

To use, call it with

`dsd-fme -fs -6 raw_signal.wav`

Just note, its going to make some large wav files, approximately 6MB per 1 Minute, or about 360 MB per hour going on a few tests I ran with it. Could still vary a bit, but expect large wav files.
